src/app/modules/submission/multirespondent-submission/multirespondent-submission.service.ts

```diff
@@ -674,25 +676,36 @@ export const performMultiRespondentPostSubmissionCreateActions = ({
   }
 
   const webhookUrl = form.webhook?.url
+
   if (webhookUrl) {
     logger.info({
-      message: 'Sending initial webhook for multirespondent submission',
+      message: 'Sending update webhook for multirespondent submission',
       meta: logMeta,
     })
 
-    WebhookFactory.sendInitialWebhook(
-      submission,
-      webhookUrl,
-      !!form.webhook?.isRetryEnabled,
-    )
-      .andThen(() => okAsync(form))
-      .mapErr((error) => {
-        logger.error({
-          message: 'Multirespondent submission webhook error',
-          meta: logMeta,
-          error,
-        })
+    if (!encryptedWebhookContent) {
+      logger.error({
+        message:
+          'Error while sending webhook, no encryptedWebhookContent found',
+        meta: { action: 'useEncryptedWebhookContentMrf' },
       })
+    } else {
+      submission.encryptedContent = encryptedWebhookContent
+
+      WebhookFactory.sendInitialWebhook(
```

Comment:
question: should this function perhaps be renamed to reflect the latest changes introduced with the encryptedWebhookContent? this is related to the comment change above